charts/opentelemetry-collector/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.50.0
+version: 0.51.0
```

```suggestion
version: 0.50.1
```
We can get away with only a patch bump for this bug fix.  Will need to re-run `make generate-examples CHARTS=opentelemetry-collector`.

---

My pleasure! That's done, thanks.